include/ebpf_structs.h

```diff
@@ -283,10 +283,7 @@ enum bpf_link_type
     BPF_LINK_TYPE_MAX
 };
 
-static const char* const _ebpf_link_display_names[] = {
-    "unspec",
-    "plain",
-};
+static const char* const _ebpf_link_display_names[] = {"unspec", "plain", "cgroup", "xdp"};
```

Comment:
Question: Is the intent that all third party extensions also have to update this list if needed?
I ask because the "xdp" extension is not in this repo, but apparently requires a change in this repo?